packages/@react-spectrum/dropzone/docs/DropZone.mdx

```diff
@@ -0,0 +1,374 @@
+  );
+}
+```
+
+The `Draggable` component used above is defined below. See [useDrag](../react-aria/useDrag.html) for more details and documentation.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example render=false export=true
+import {useDrag} from '@react-aria/dnd';
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world',
+        'my-app-custom-type': JSON.stringify({message: 'hello world'})
+      }];
+    }
+  });
+
+  return (
+    <div {...dragProps} role="button" tabIndex={0} className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+```
+</details>
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.draggable {
+  display: inline-block;
+  vertical-align: top;
+  border: 1px solid gray;
+  padding: 10px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  border-radius: 4px;
+  height: fit-content;
+}
+
+.draggable.dragging {
+  opacity: 0.5;
+}
+
+.files{
+  margin-top: 20px;
+}
+```
+
+</details>
+
+
+## Props
+
+<PropTable component={docs.exports.DropZone} links={docs.links} />
+
+## Visual options
+
+### Filled state
+
+The user is responsible for both managing the filled state of a drop zone and handling the associated styling. To set the drop zone to a filled state, the user must pass the `isFilled` prop.
+
+The example below demonstrates one way of styling the filled state.
+
+```tsx example
+function Example() {
+  let [filledSrc, setFilledSrc] = React.useState(null);
+  let [isFilled, setIsFilled] = React.useState(false);
+
+  return (
+    <>
+      <DraggableImage />
+      <DropZone
+        isFilled={isFilled}
+        maxWidth="size-3000"
+        height="size-2400"
+        UNSAFE_className={isFilled && 'is-filled'}
+        getDropOperation={(types) =>  (types.has('image/png') || types.has('image/jpeg')) ? 'copy' : 'cancel'}
+        onDrop={async (e) => {
+          e.items.find(async (item) => {
+            if (item.kind === 'file') {
+              if (item.type === 'image/jpeg' || item.type === 'image/png') {
+                setFilledSrc(URL.createObjectURL(await item.getFile()));
+                setIsFilled(true);
+              }
+            } else if (item.kind === 'text') {
+              setFilledSrc(await item.getText('image/jpeg'));
+              setIsFilled(true);
+            }
+          });
+        }}>
+        <IllustratedMessage>
+            <Upload />
+            <Heading>
+              <Text slot="label">
+                Drag and drop photos
+              </Text>
+            </Heading>
+        </IllustratedMessage>
+        {isFilled && <img className={'images'} alt="" src={filledSrc} />}
+      </DropZone>
+    </>
+  );
+}
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.images{
```

Comment:
style nit, I'm surprised this doesn't get formatted automatically (I guess dprint just does js(x))

```suggestion
.images {
```